config.yaml

```diff
@@ -343,3 +343,9 @@ options:
       respond with appropriate authentication details. For more info, please refer
       to the upstream documentation at
       https://kubernetes.io/docs/reference/access-authn-authz/authentication/#webhook-token-authentication
+  pod-security-policy:
+    type: string
+    default: ""
+    description: |
+      Default RBAC pod security policy and privileged cluster roles.
+      https://kubernetes.io/docs/concepts/policy/pod-security-policy/
```

Comment:
With strings used for a lot of different things, you should probably clarify what the expected input is. Do you expect 'yaml file as string', or base64 encoded file, or something else?